.github/CONTRIBUTING.md

```
@@ -118,31 +188,57 @@ If there isn't an existing DESIGN.md file:
 
 ## Requesting Review
-Once your PR is ready, assign reviewers and mark it as "ready to review". But before that, make sure to report the time you have spent on the PR.
+
+Once your PR is ready, assign reviewers and mark it as "ready to review." But before that, report the time you have spent on it.
 
 > [!NOTE]
-> When contributing, it's essential to report time accurately, including all stages of development (planning, implementation, QA). We encourage opening a PR at the start of your work, even during the planning or investigation phase. Programming and designing isn't just about writing code or creating designs; it also involves planning (40%) and QA (20-30%). 
-> 
+> When contributing, it's essential to report time accurately, including all stages of development (planning, implementation, QA). We encourage opening a PR at the start of your work, even during the planning or investigation phase. Programming and designing isn't just about writing code or creating designs; it also involves planning (40%) and QA (20-30%).
 
 #### Code Quality and Reviews
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)